CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -113,12 +113,16 @@ local function UpdateLayout()
 		end
 	end
 	
-	width = #ToolsList * ICON_SPACING
-	height = ICON_SIZE + HEALTHBAR_SPACE
+	if #ToolsList == 0 then
```

Comment:
I see, yeah in the future we should probably rename this script, its basically top bar for vr